### DISCUSSION TITLE
AP college board extension exploit

### DISCUSSION THREAD

**Discussion by Gagnastiak** [3kh0/ext-remover#217] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
For a temporary unblock for all sites just use this hack (doesn't run Geforce now). Sign out of your chromebook. In the left hand corner you will see the apps button. Click on it and go to AP college board. Click sign in as teacher and it will redirected you to another page after it. Once you get there, click on student in the top right corner. Scroll all the way to the bottom and then click on the Youtube icon. From there, you can access Youtube, or you can search for "Google link" and a video will have the google link in the description. You then can search any site using that link.

**Comment by ghost** [comment]
when you go to youtube, hit sign in. Dont sign in, but click the small privacy button. Scroll all the way to the bottom and click google. You will then have every site unblocked

**Reply by Theblackmamba824** [reply]
r u trolling

**Reply by ghost** [reply]
I mean youtube in the kiosk mode

**Reply by Theblackmamba824** [reply]
kiosk mode?

**Reply by ghost** [reply]
when youi open the college board...

**Reply by Aka-but-nice** [reply]
These kids don't know what a kiosk app is💀

**Reply by Alex2Mayhem** [reply]
Bro doesn't know what a kiosk is? 💀💀💀💀

**Reply by ghost** [reply]
lol

**Comment by rwg100** [comment]
kiosk apps are exclusive to school disctricts. if your in a different one this wont work

**Reply by fnfgo** [reply]
i think they control+alt+deleted college board. :(

**Reply by CheezeKake77** [reply]
As far as I know only high schools use this

**Comment by cooltoo37** [comment]
when you go to college  board just create account then press on student on top right corner then go down all the way and press contact us then click on any email address there and then there is a dotted box just like google press on that then scroll till you see google then click on that here is a video to explain it better https://www.youtube.com/shorts/xkzDQO9H7hE  credits to https://www.youtube.com/@IrwinTech for finding it go check his youtube he has more stuff.

**Reply by cooltoo37** [reply]
sorry left out some steps check the yt link for full instructions
